oggm/core/flowline.py

```diff
@@ -237,7 +237,7 @@ def __init__(self, line=None, dx=None, map_dx=None, surface_h=None,
         if np.any(self._w0_m <= 0):
             raise ValueError('Trapezoid beds need to have origin widths > 0.')
 
-        self._prec = np.where(lambdas == 0)
+        self._prec = np.where(lambdas == 0)[0]
```

Comment:
Wait, but `TrapezoidalBedFlowline` is never used operationally.

Reply:
We use MixedBedFlowline if I recall well!

Reply:
yes, I just added this on the fly to get an array and not a tuple(array). Not strictly necessary for the TrapezoidalBed but for the MixedBed. I could account for this at the time where I change the attribute values if you prefer, but I think it is nicer to have the actual array as an attribute like it is true for ['_ptrap']

Reply:
No, that's fine. You probably are more into it than I am currently.